source/find_getmore_killcursors_commands.rst

```diff
@@ -547,6 +555,11 @@ The accepted parameters are described in the table below.
      - If not set, the server defaults to it’s internal maxTimeMS setting.
 
        Please see the "Semantics of maxTimeMS" section for more details.
+   * - comment
+     -
+     - any
+     - Enables users to specify an arbitrary comment to help trace the operation through the database profiler, currentOp and logs. The default is to not send a value.
```

Comment:
It sounds very reasonable. I think I can still imagine a situation when one may want to override comment on `getMore` command; however, it would be a very artificial situation.

I changed specs accordingly, and added tests.